src/v2/view-builder/views/TerminalView.js

```diff
@@ -94,8 +94,6 @@ const Body = BaseForm.extend({
       messagesObjs.value.push({ message: loc('oie.consent.enduser.email.allow.description', 'login')});
     } else if (this.options.appState.containsMessageWithI18nKey('tooManyRequests')) {
       description = loc('oie.tooManyRequests', 'login');
-    } else if (this.options.appState.containsMessageWithI18nKey('security.access_denied')) {
```

Comment:
Does removing this render the error correctly on the terminal pages ? Can you add a screenshot for terminal and non terminal views ?

Reply:
let me grab it

Reply:
update with screenshot